tools/nd_freeze_helper.pl

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/perl
+# Simple web client for retrieving neurodebian snapshot information.
+# Code adapted from: https://www.oreilly.com/openbook/webclient/ch04.html
```

Comment:
oy... what was the license for original code if it was used largely to produce this one?
Note that a little disadvantage of a separate script is that users might need to download two scripts now.
BUT cool that we can do that. Also later I might ask to add an option so it doesn't parse anything and just dumps output into a specified file? then I could use it instead of wget in other places